src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: {
+  title,
+  body,
+  user,
+  comments,
+} }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
```

Comment:
the `user` might be `null`
in this case `UserInfo` component doesn't make sense. Moreover it will lead to an error because you will try to get values from `null`

Reply:
still not fixed
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/55200530/bd10a1b7-63e5-4ac4-985a-35d5a62f50b7)